h/groups/util.py

```diff
@@ -21,3 +21,15 @@ def __acl__(self):
             (security.Allow, 'authority:{}'.format(self.auth_domain), 'write'),
             security.DENY_ALL,
         ]
+
+    @property
```

Comment:
There is a comment above about this group only implementing `__acl__` which needs updating, if these properties are considered part of the interface that groups must implement.